pkg/util/trace.go

```diff
@@ -0,0 +1,23 @@
+package util
+
+import (
+	"log"
+
+	"contrib.go.opencensus.io/exporter/ocagent"
+	"go.opencensus.io/trace"
+)
+
+// InitialiseTracing initialises trace, exporter and the sampler
+func InitialiseTracing(serviceName string, address string, probability float64) {
+	oce, err := ocagent.NewExporter(
+		ocagent.WithInsecure(),
+		ocagent.WithAddress(address),
```

Comment:
What happens when the address flag is not provided?  It looks like the default value of "" is provided here, which seems to turn off tracing as we would want.  A bit surprising that this doesn't trigger the fatal error below though.  A safer thing might be to just return right away if the address is "".